CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+
+#### unreleased - upcoming
+
+- Add `requireUserWithAnyRole` and `anyRoleExcept`, to be used in the
```

Comment:
I think that call center permissions are used in a variety of ways. They are not connected to features in the software but to the actual roles, so we have rules like "analysts should not see the calls list", which makes the calls list visible to any role except the analyst role. So basically we don't have permissions, but rather roles